### ISSUE TITLE
Tech Weekly or DN: Weekly

### ISSUE THREAD

**Issue by bibstha** [developers-nepal/Bagaincha#9]
I'm a big fan of Ruby Weekly (I get all my latest news about things happening around Ruby from there). And a small fan of PHPWeekly (I do not check it that often but I once in a while to know the status of PHP7 and the new frameworks).

There are a lot of other Weeklies out there, JSWeekly, HTML5Weekly etc and they seem to be very popular. 

I see the advantage of these weekly is to keep all the developers upto date about a topic.

Since we have the **Developer Nepal** movement going on, I'd like to propose the idea that we have a **DeveloperNepal Weekly** or **DN: Weekly**. This includes a roundup of 
- Blog posts on Tech from Nepali writers
- Events, JSMeetup, PHPMeetup, RubyMeetup, JAVAMeetup (if that exists), DataNerdMeetup, etc etc.
- JobPostings (This can be sponsored to have the people working on the Weekly some pocket money to stay motivated)

I myself can't take the lead on this due, but am willing to help setup such a stuff if anyone is interested.

**Comment by geshan** [comment]
good idea, any thoughts @hardfire @roshanbh @prashishh @nootanghimire @developers-nepal/php ?

**Comment by hardfire** [comment]
I would say it is an amazing idea. Curious if there are any takers or if anyone wants to initiate this. 

If not, then I could suggest that we can spend some time to make the idea more concrete and see if we could hire(not really the best word here) someone to do this, should not be so tough to get some sponsorship from the community, for the community :D

**Comment by techgaun** [comment]
:+1: 
@bibstha @geshan @hardfire This is an amazing idea. I can sponsor domain and mailchimp/sns. We can use github page to host the weekly page. Do we have css/html ninja out there to help?

**Comment by geshan** [comment]
Rather than domain etc we need people ready to volunteer on a weekly basis
initially. Then after it gains momentum we can think about domain etc.

Warm Regards,
Geshan Manandhar

**Comment by hardfire** [comment]
Exactly, what is required is someone who can take lead. maybe to start with
we can do it bi-weekly and call it DN Fortnightly :D and then i think we
can together curate it, if there are two weeks. To start with, I can do it
but would like one more person giving a hand to curate and stuff.

On 28 August 2015 at 08:07, Geshan Manandhar notifications@github.com
wrote:

> Rather than domain etc we need people ready to volunteer on a weekly basis
> initially. Then after it gains momentum we can think about domain etc.
> 
> Warm Regards,
> Geshan Manandhar
> On Aug 28, 2015 10:01 AM, "Samar Dhwoj Acharya" notifications@github.com
> wrote:
> 
> > [image: :+1:]
> > @bibstha https://github.com/bibstha @geshan <https://github.com/geshan
> > 
> > @hardfire https://github.com/hardfire This is an amazing idea. I can
> > sponsor domain and mailchimp/sns. We can use github page to host the
> > weekly
> > page. Do we have css/html ninja out there to help?
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/developers-nepal/Bagaincha/issues/9#issuecomment-135643016
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/developers-nepal/Bagaincha/issues/9#issuecomment-135643482
> .

## 

---

Avinash Kundaliya
avinash@avinash.com.np
http://avinash.com.np

**Comment by bibstha** [comment]
How about we create an Issue in github for one edition of the weekly. And people share the links as comments? We can compile them easily then into a newsletter.

**Comment by geshan** [comment]
Curating can be done with a group email. We need 2 guys to take lead
currently I can't volunteer :)

Warm Regards,
Geshan Manandhar

**Comment by hardfire** [comment]
Right, I think that one more person should come up and then we can kick-off something. Managing it wouldn't be hard, need someone to take lead though.

**Comment by bibstha** [comment]
To begin with, I'll propose a structure for the weekly.

We could have these sections
1. Major News
   - PHP7 release announced or Ruby or JS EMCA etc announcements.
2. Meetups and Events
   - JS / PHP / Ruby, for whom, when and where, with links to their pages
   - Hackathons, Code challenges etc
3. Job and Internship opportunities
4. Picks
   - A small (one or two link each from everyone) to be shared with the community. This could be some nice tutorial page, online games, a special course in coursera useful for the students etc.

Let in the feedbacks.

**Comment by hardfire** [comment]
Yep, I'd suggest one sub-section on Startups. news from the startups/entrepreneurship front of Nepal.

**Comment by bibstha** [comment]
@hardfire :+1:

**Comment by techgaun** [comment]
Well, I would be happy to volunteer on this.

**Comment by hardfire** [comment]
awesome! then let's talk more @techgaun about how we want to structure this and take it out. Lets plan to release the first DN:Fortnightly on Bhadra 31 or Ashwin 1. Which gives us a little more than 15 days for the first one. should be fine, i guess. Lets continue the discussion for the first one at https://github.com/developers-nepal/Bagaincha/issues/10

**Comment by bibstha** [comment]
I've started posting things for our fortnightly1 [here](https://github.com/developers-nepal/Bagaincha/issues/10)
